types/node/globals.d.ts

```diff
@@ -62,6 +62,9 @@ interface AbortSignal {
      * Returns true if this AbortSignal's AbortController has signaled to abort, and false otherwise.
      */
     readonly aborted: boolean;
+
+    onabort: ((this: AbortSignal, event: Event) => any) | null;
```

Comment:
Can we change `any` to `void`?

Reply:
Also the `Event` class/interface is not yet defined in node typings.

Reply:
![Screenshot 2021-08-21 at 18 21 10](https://user-images.githubusercontent.com/14818450/130328378-9dacf843-fa34-4c51-a889-544fcc902db6.png)

I picked any, so its compatible with lib.dom definition, is it save to use void?